network-upgrader/Cargo.toml

```diff
@@ -15,9 +15,16 @@ path = "src/main.rs"
 
 [dependencies]
 anyhow = "1.0"
+async-trait = "0.1"
+aws-config = { version = "1.1.7", features = [
+  "behavior-version-latest",
+] }
+aws-sdk-kms = "1.37.0"
+k256 = { version = "0.13.3", features = ["ecdsa-core"] }
 clap = { version = "4.5", features = ["derive", "env"] }
 fuel-tx = "0.56.0"
-fuels = "0.66.0"
+# This branch is a temporary solution because fuels has froze breaking changes merge until december
+fuels = { git = "https://github.com/FuelLabs/fuels-rs.git", branch = "move_method_from_account_to_view_only_account" }
```

Comment:
We should open an issue to track this

Reply:
Done in #5

Reply:
It would be nice to use `patch` section instead of using branch=)